seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/oceanbase/OceanBaseOracleCatalog.java

```diff
@@ -53,6 +52,21 @@ protected String getListDatabaseSql() {
         throw new UnsupportedOperationException();
     }
 
+    @Override
+    protected String getDatabaseWithConditionSql(String databaseName) {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
Hi @dailai  ,Why oracle support `getDatabaseWithConditionSql` but OceanBaseOracle can not support `getDatabaseWithConditionSql`?

Reply:
> Hi @dailai ,Why oracle support `getDatabaseWithConditionSql` but OceanBaseOracle can not support `getDatabaseWithConditionSql`?

Beacase this method named getListDatabaseSql is not supported.